### PR TITLE
Remove old variables, add our more modern style of overrides

### PR DESCRIPTION
**Pull request by jibsheet**

### CODE REVIEW

**Review by michaelroytman — APPROVED** [review]

**Comment by jibsheet** [comment]
@thallada would you mind double checking the variable deletions here?  It looks like Jarvis pulled them out years ago, but I always like to check.

**Review by thallada — APPROVED** [review]
I checked that we don't use those variables in the Analytics API config anymore. I'm assuming the real ANALYTICS_API_DEFAULT_* and ANALYTICS_API_REPORTS_* values are being defined in the secure config repo?

**Comment by jibsheet** [comment]
@thallada Yeah, right now secure is overriding all of ANALYTICS_API_DATABASES: I'm trying to just override the passwords and then putting the database name, username and hostname in the internal config where folks can see it.  But that was hard to do when overriding the whole structure.

File: CHANGELOG.md [hunk 1]
```diff
@@ -330,4 +330,8 @@
 - Role: insights
   - Removed `bower install` task
   - Replaced r.js build task with webpack build task
-  - Removed `./maange.py compress` task
\ No newline at end of file
+  - Removed `./maange.py compress` task
```
Comment (thallada) [inline CHANGELOG.md]: Nit: If you could fix my spelling mistake, that'd be great maan.

File: CHANGELOG.md [hunk 2]
```diff
@@ -330,7 +330,11 @@
 - Role: insights
   - Removed `bower install` task
   - Replaced r.js build task with webpack build task
-  - Removed `./maange.py compress` task
+  - Removed `./mange.py compress` task
```
Comment (feanil) [inline CHANGELOG.md]: Mange?  I don't think we want to give the software mange.

**Review by feanil — APPROVED** [review]

**Review by feanil — APPROVED** [review]

**Pull request merged**